CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 Changelog
 =========
+
+# 0.6.0 (2019-02-19)
+
+## New Features
+
+- **M3DB** (Config): Simplify M3 config options (#1371)
+- **M3Coordinator**: Improvements to database creation API (#1350)
+- **M3Coordinator**: Enforce ID generation scheme explicitly added to configs (#1381)
+- **M3Query**: Add quantile_over_time (#1367) and histogram_quantile (#1372) Prometheus functions
+- **Documentation**: Additional documentation for namespace setup and configuration (#1350), etcd (#1354), and M3Coordinator ID generations schemes (#1381, #1385)
+
+## Performance
+
+- **M3DB** (Index): Add posting list cache (#1370) that should result in a massive improvement for M3DB query performance for most workloads
+
+## Bug fixes
+
+- **M3DB** (Index): Fix to race condition in index query (#1356)
+- **M3Coordinator**: Fix to panic responder which was previously not reporting query panics to users (#1353)
```

Comment:
Also fix the temporal issues (https://github.com/m3db/m3/issues/1344)?

Reply:
That was already fixed in 0.5.0; should we retroactively mention it there instead?

Reply:
As discussed, was introduced post 0.5.0